config.go

```diff
@@ -98,6 +99,9 @@ const (
 	// HTLCs on our channels.
 	minTimeLockDelta = routing.MinCLTVDelta
 
+	// MaxTimeLockDelta is the maximum timelock.
```

Comment:
nit: more precise comment? Perhaps something along the lines of "... is the maximum CLTV delta that can be applied to forwarded HTLCs"?